haralyzer/assets.py

```diff
@@ -632,3 +632,84 @@ def video_load_time(self):
     @cached_property
     def html_load_time(self):
         return self._get_asset_load('html')
+
+
+class HarEntry(object):
+    """
+        An object that represent one entry in a HAR Page
+    """
+    def __init__(self, entry):
+        self.raw_entry = entry
+
+    def get_header_value(self, name):
+        """
+        Returns the header value of the header defined in ``name``
+
+        :param name: ``str`` name of the header to get the value of
+        """
+        for x in self.headers:
+            if x["name"].lower() == name.lower():
+                return x["value"]
+
+    @cached_property
+    def headers(self):
+        return self.raw_entry["request"]["headers"] + self.raw_entry["response"]["headers"]
+
+    @cached_property
+    def request(self):
+        return Request(entry=self.raw_entry["request"])
+
+    @cached_property
+    def response(self):
+        if type(self.raw_entry) == dict:
+            return Response(entry=self.raw_entry["response"])
+        return self.raw_entry.response
+
+    @cached_property
+    def startTime(self):
+        try:
+            return parser.parse(self.raw_entry["startedDateTime"])
+        except KeyError:
+            return
+
+    @cached_property
+    def time(self):
+        return self.raw_entry["time"]
+
+    @cached_property
+    def pageref(self):
+        return self.raw_entry["pageref"]
+
+    @cached_property
+    def server_address(self):
+        return self.raw_entry["serverIPAddress"]
+
+    @cached_property
+    def port(self):
+        return int(self.raw_entry["connection"])
+
+    @cached_property
+    def cookies(self):
+        try:
+            return self.raw_entry["cookies"]
+        except KeyError:
+            return []
+
+    @cached_property
+    def secure(self):
+        try:
+            return self.raw_entry["_securityState"] == "secure"
```

Comment:
I think you could avoid the `try/except`, like `return self.raw_entry.get('_securityState', '') == 'secure'`